src/App.tsx

```diff
@@ -5,6 +5,10 @@ import { ImportModal, Header } from "./Components/";
 const App: React.FC = () => {
   const [files, setFiles] = useState<File[]>([]);
   const [show, setShow] = useState<boolean>(false);
+  const [totalVideoDuration, setTotalVideoDuration] = useState<number>(0);
+  // mapping of file index to original duration of video/images
+  const [oriDur, setOriDur] = useState<{ [fileindex: number]: number }>({});
```

Comment:
This information should live alongside all the file data. Again, TBD in our follow-up conversation (mentioned in #6 )